src/app/offers/offers.component.ts

```diff
@@ -42,15 +44,17 @@ export class OffersComponent implements OnInit, OnDestroy {
       let baseCurrencyCodes = _.map(this.offerList, 'baseCurrencyCode');
       let currencyCodes = _.map(this.offerList, 'currencyCode');
       let counterCurrencyCodes = _.map(this.offerList, 'counterCurrencyCode');
-      
+
       this.currencyCodes = _.uniq([...baseCurrencyCodes, ...currencyCodes, ...counterCurrencyCodes]);
       return this.marketPriceService.getMarketPrices(true, this.currencyCodes);
     }).then(marketPrices => {
       this.prices = marketPrices;
       this.loading = false;
     }).catch(error => {
-      this.toast.error(error, 'TOAST.OFFERS.CANT_FETCH_DATA');
-      this.loading = false;
+      if (this.settings.backendUrl) {
```

Comment:
Instead of masking the error you shouldn't call `refreshOffersList` if backendUrl is not set.